CONTRIBUTING.md

```diff
@@ -102,8 +102,14 @@ This package contains React components to perform requests in a declarative way.
 
 Once it's done, you can run `yarn start` or `yarn test` (`yarn test:watch`) to develop the packages.
 
+## Developing locally
+
+To develop locally, you can use the `playground` application to test the changes in some of the packages. Make sure to `yarn build` the packages before starting the `playground` app because the app consumes the packages as normal "transpiled" dependencies.
```

Comment:
Maybe mentioning `build watch:es` makes sense at least that's what you'd want to end up doing loads of times I think.